util/dHomogTransInv.m

```diff
@@ -20,4 +20,7 @@
 ret = zeros(numel(T), nq);
 ret = setSubMatrixGradient(ret, dinvT_R, 1:3, 1:3, size(T));
 ret = setSubMatrixGradient(ret, dinvT_p, 1:3, 4, size(T));
-end
\ No newline at end of file
+
+ret(16, :) = 1;
```

Comment:
Oh :-( sorry I thought this was just the Homogeneous Transform.  I will fix
this.

-Andy S.


On Thu, Jul 30, 2015 at 11:21 PM, Twan Koolen <notifications@github.com>
wrote:

> In util/dHomogTransInv.m
> <https://github.com/RobotLocomotion/drake/pull/1212#discussion_r35944060>:
>
> > \ No newline at end of file
> > +
> > +ret(16, :) = 1;
>
> wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1212/files#r35944060>.
>